content/en/_index.md

```diff
@@ -142,14 +143,46 @@ function example_snippet() {
 }
 // [END product_example]
 {{< /tab >}}
+
+{{< tab header="Terraform" >}}
+// Terraform doesn't have `import` statements. However, Terraform
+// has the concept of importing resources to bring them under
+// Terraform management. For more information, see
+// [Import your Google Cloud resources into Terraform state](https://cloud.google.com/docs/terraform/resource-management/import).
```

Comment:
Hyperlink doesn't render in the tab view

Reply:
Removed content